### PR TITLE
fix bug in DeleteContext when index equals to zero

### PR DESCRIPTION
**Pull request by adi88d**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16485

## Description
fix bug in DeleteContext - the scrip refer index param as undefined when it equals to zero

### CODE REVIEW

File: Scripts/script-DeleteConext.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ script: |
           ReadableContentsFormat: formats.markdown,
           EntryContext: keysToKeepObj
       };
-  } else if (args.index) {
+  } else if (args.index != undefined)) {
```
Comment (glicht) [inline Scripts/script-DeleteConext.yml]: Best to use strict compare
```suggestion
  } else if (args.index !== undefined)) {
```

Comment (glicht) [reply Scripts/script-DeleteConext.yml]: Never mind. For undefined strict compare is not needed.

**Comment by anara123** [comment]
@adi88d fix the conflict

File: Tests/conf.json [hunk 2]
```diff
@@ -1385,7 +1472,13 @@
         "JoeSecurityTestDetonation": "command joe-download-report fails (issue 16118)",
         "af2f5a99-d70b-48c1-8c25-519732b733f2": "Added here because test existed but was not configured - need to review the test",
         "Zoom_test": "Added here because test existed but was not configured - need to review the test",
-        "Test - CrowdStrike Falcon": "Test is unmockable, and has its data deleted every few weeks"
+        "Test - CrowdStrike Falcon": "Test is unmockable, and has its data deleted every few weeks",
+        "block_endpoint_-_carbon_black_response_-_test": "Test for deprecated integration",
+        "get_file_sample_from_path_-_generic_-_test": "Test for deprecated integration",
+        "get_file_sample_from_path_-_carbon_black_enterprise_response_-_test": "Test for deprecated integration",
```
Comment (anara123) [inline Tests/conf.json]: if the integration is deprecated, we can delete the test

**Pull request merged**